suites/pacific/rbd/tier-1_rbd.yaml

```diff
@@ -19,6 +19,8 @@ tests:
       abort-on-fail: true
       module: install_prereq.py
       name: install ceph pre-requisites
+      config:
```

Comment:
is this required to add is_production : True?

Reply:
Removed